Update main readme (#912)
netgroup-polito/CrownLabs: README.md

CrownLabs relies on two major high-level components:
* **The Backend Business Logic**, which provides the different CrownLabs functionalities and is implemented by custom Kubernetes operators (e.g. the Instance Operator);

> * **qlkube**, a middleware that enables the exposition of the Kubernetes API Server as a GraphQL service;

Reviewer: ```suggestion
* **qlkube**, a middleware that enables to expose the Kubernetes API Server as a GraphQL service;
```
